src/components/share/ShareDirective.js

```diff
@@ -69,13 +81,47 @@
                 }).success(function(response) {
                   scope.permalinkValue = response.shorturl;
                   scope.urlShortened = true;
+                  scope.$applyAsync(function() {
+                    // Auto-select the shortened permalink (not on mobiles)
+                    if (!gaBrowserSniffer.mobile) {
```

Comment:
just here, this applyAsync is still needed in order to keep autoselect applied after ajax success